perun-core/src/main/java/cz/metacentrum/perun/core/entry/MembersManagerEntry.java

```diff
@@ -846,6 +846,20 @@ public RichMember getRichMemberWithAttributes(PerunSession sess, Member member)
 		return getPerunBl().getMembersManagerBl().filterOnlyAllowedAttributes(sess, getMembersManagerBl().getRichMembersWithAttributes(sess, vo, status), null, true);
 	}
 
+	@Override
+	public List<RichMember> getRichMembersNoUserAttributes(PerunSession sess, Vo vo) throws PrivilegeException, VoNotExistsException {
+		Utils.checkPerunSession(sess);
+
+		getPerunBl().getVosManagerBl().checkVoExists(sess, vo);
+
+		// Authorization
+		if (!AuthzResolver.authorizedInternal(sess, "getRichMembers_Vo_policy", vo)) {
```

Comment:
You should also add the new method to the `perun-roles.yml`. The yml definition will be the same as `getRichMembers_Vo_policy`.
```suggestion
		if (!AuthzResolver.authorizedInternal(sess, "getRichMembersNoUserAttributes_Vo_policy", vo)) {
```